docs/src/main/paradox/postgresql-cdc.md

```diff
@@ -0,0 +1,152 @@
+
+Example source settings:
+
+
+Scala
+: @@snip ($alpakka$/postgresql-cdc/src/test/scala/akka/stream/alpakka/postgresqlcdc/TestScalaDsl.scala) { #SourceSettings }
+
+Java
+: @@snip ($alpakka$/postgresql-cdc/src/test/java/akka/stream/alpakka/postgresqlcdc/TestJavaDsl.java) { #SourceSettings }
+
+
+
+### Source Usage
+
+Without further ado, a minimalist example:
+
+Scala
+: @@snip ($alpakka$/postgresql-cdc/src/test/scala/akka/stream/alpakka/postgresqlcdc/TestScalaDsl.scala) { #GetExample }
+
+Java
+: @@snip ($alpakka$/postgresql-cdc/src/test/java/akka/stream/alpakka/postgresqlcdc/TestJavaDsl.java) { #GetExample }
+
+
+### Ack Sink Settings
+
+We configure the Ack sink using @scaladoc[PostgreSQLInstance](akka.stream.alpakka.postgresqlcdc.PostgreSQLInstance) and @scaladoc[PgAckSinkSettings](akka.stream.alpakka.postgresqlcdc.PgAckSinkSettings):
+
+@scaladoc[PostgreSQLInstance](akka.stream.alpakka.postgresqlcdc.PostgreSQLInstance)
+
+|Setting                 | Meaning                           | Required|
+| -----------------------|-----------------------------------|---------|
+| JDBC connection string | JDBC connection string            | yes     |
+| slot name              | name of logical replication slot  | yes     |
+
+@scaladoc[PgAckSinkSettings](akka.stream.alpakka.postgresqlcdc.PgAckSinkSettings):
+
+|Setting               |Meaning                                                                     | Default              | Required |
+|----------------------|----------------------------------------------------------------------------|----------------------|----------|
+| maxItems             | ideal number of items to acknowledge at once                               | 16                   | no       |
+| maxItemsWait         | maximum duration for which the stage waits until maxItems have accumulated | 3 second             | no       |
+
+
+### Ack Sink Usage
+
+You want to map these database change events (i.e. RowInserted) to real domain events
+(i.e. UserRegistered - aiming to adopt a Domain Driven Design approach) and publish the domain events to a queue (e.g, AWS SQS)
+and use the Ack Sink to acknowledge consumption.
+
+Scala
+: @@snip ($alpakka$/postgresql-cdc/src/test/scala/akka/stream/alpakka/postgresqlcdc/TestScalaDsl.scala) { #PeekExample }
+
+Java
+: @@snip ($alpakka$/postgresql-cdc/src/test/java/akka/stream/alpakka/postgresqlcdc/TestJavaDsl.java) { #PeekExample }
+
+
+## Limitations
+
+* It cannot capture events regarding changes to tables' structure (i.e. column removed, table dropped, table
+index added, change of column type etc.).
+* When a row update is captured, the previous version of the row is lost / not available.
+    * Unless the table has a non-default `REPLICA IDENTITY` setting. To get the full previous version of the row: `ALTER TABLE "TABLE_NAME" REPLICA IDENTITY FULL`.
+* It doesn't work with older version of PostgreSQL (i.e < 9.4).
+
+## Important Note
+
+* Remember to drop the slot after you are done using it.
```

Comment:
Could you recommend a way to accomplish it?
On stream completion, or Actor system termination?
Maybe a Paradox [warning](https://developer.lightbend.com/docs/paradox/current//directives/callouts.html#warning-callout) is what you need here.